gradle/java.gradle

```diff
@@ -11,7 +11,7 @@ compileJava {
 }
 
 task testJar(type: Jar, dependsOn: testClasses, group: "build") {
-    appendix = 'test'
+    classifier = 'test'
```

Comment:
:+1: btw: the file `h2o-algos/testMultiNode.sh` references test jar but it should be fine:

```
JVM="nice $JAVA_CMD $COVERAGE -ea -Xmx${MAX_MEM} -Xms${MAX_MEM} -DcloudSize=4 -cp build/libs/h2o-algos-test.jar${SEP}build/libs/h2o-algos.jar${SEP}../h2o-core/build/libs/h2o-core-test.jar${SEP}../h2o-core/build/libs/h2o-core.jar${SEP}../h2o-genmodel/build/libs/h2o-genmodel.jar${SEP}../lib/*"
```